# Engine id
string id

# Attribute name to get
string name
---
# Attribute value (empty string if failed)
string attribute
